en/components/sprite.md

```diff
@@ -33,7 +33,13 @@ Currently, the Sprite component supports five rendering modes:
 - `Simple mode`: rendering the Sprite according to the original image resource. It is normally used along with `Use Original Size` to guarantee the image shown in the scene is in full accordance with the image designed by the graphic designer.
 - `Sliced mode`: the image is cut up into a nine square grid and according to certain rules is scaled to fit freely set dimensions (`size`). It is usually used in UI elements or to make images that can be enlarged infinitely without influencing the image quality into images cut up into a grid to save game resource space. Please read [use Sprite editor to make a Sudoku image](../asset-workflow/sprite.md#-sprite-) for detailed information.
 - `Tiled mode`: as the size of the Sprite increases, the image is not stretched, but it Will repeat according to the size of the original picture, paving the original picture to the size of the entire Sprite as tiled tiles.
+
  ![tiled](sprite/tiled.png)
+
+  Sprites in Tiled mode will be affected by the division of nine grids. At this time, each picture will be cut according to the `insetTop`, `insetBottom`, `insetLeft`, and `insetRight` properties recorded by the SpriteFrame resource object, and the edges will be stretched and the center tile will be expanded.
```

Comment:
A Sprite in Tiled mode will be affected by its Sliced config. It will be sliced according to the `insetTop`, `insetBottom`, `insetLeft`, and `insetRight` properties recorded in the SpriteFrame. The corners will remain unscaled when the Sprite is stretched, while the center sections will be repeated instead of stretched.